cpu/kinetis/Makefile.features

```diff
@@ -21,4 +21,9 @@ else
 FEATURES_PROVIDED += periph_mcg
 endif
 
+ifneq (,$(filter W K,$(KINETIS_SERIES)))
```

Comment:
@aabadie while the dependency issue issn't fixed, I can work around with filtering by `CPU_MODEL`